src/lseq/ident.rs

```diff
@@ -255,39 +262,55 @@ impl<A: Actor> IdentGen<A> {
     }
 }
 
-#[cfg(test)]
```

Comment:
We now package up `impl Arbitrary for _` with the library for users to perform their own property based testing.